contracts/proxies/StakingProxy.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity ^0.4.24;
+
+import "../Staking.sol";
+import "./ThinProxy.sol";
+import "@aragon/os/contracts/lib/token/ERC20.sol";
+
+
+contract StakingProxy is ThinProxy {
+    // keccak256("aragon.network.staking")
+    bytes32 internal constant IMPLEMENTATION_SLOT = 0xbd536e2e005accda865e2f0d1827f83ec8824f3ea04ecd6131b7c10058635814;
+
+    string internal constant ERROR_WRONG_IMPLEMENTATION_SLOT = "STAKING_PROXY_WRONG_IMPL_SLOT";
```

Comment:
Can't find where this is used

Reply:
Yea, this is old stuff, will remove it, thx!